packages/flame/lib/src/components/position_component.dart

```diff
@@ -132,9 +142,9 @@ class PositionComponent extends Component
   /// rotated around its anchor point in the clockwise direction if the
   /// angle is positive, or counterclockwise if the angle is negative.
   @override
-  double get angle => transform.angle;
+  double get angle => transform.angle - angularOffset;
```

Comment:
This should not take `angularOffset` into regards.

Reply:
Modified all the angle calculations to not use this new property.